src/constructors.jl

```diff
@@ -103,3 +103,24 @@ end
 function Base.call{T, N}(::Type{NullableArray{T, N}})
     NullableArray(T, ntuple(i->0, N))
 end
+
+#----- Conversion from Array of Nullables #1 -----------------------------------#
+# Take an Array of Nullable objects and convert it into a NullableArray
+function Base.convert{S,T,N}(::Type{NullableArray{S,N}},in::Array{Nullable{T},N})
+   out = NullableArray{S,N}(Array(S,size(in)),Array(Bool,size(in)))
+   for i = 1:length(in)
+       (out.isnull[i] = in[i].isnull) ? nothing : out.values[i] = in[i].value
```

Comment:
Yes, it was weird but I it was the fastest code I could think of. I still don't see an `unsafe_get` or similar method for getting when you know it's not null, even if `in[i]` was copied locally...